.github/workflows/reassurePerformanceTests.yml

```diff
@@ -0,0 +1,45 @@
+name: Reassure Performance Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ['**']
```

Comment:
I think we should do:

```
  pull_request:
    types: [opened, synchronize]
    branches-ignore: [staging, production]
```

We do not want to run on those branches as they are used in the deploy process.